### PR TITLE
Correct the "lambda/function" formatting

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Correct the formatting of "lambda" and "function" terms according to the guidelines
- Provide small fixes

**Related issue(s)**
#1753

### CODE REVIEW

File: docs/event-bus/docs/001-overview-event-bus.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ title: Overview
 
 Kyma Event Bus enables the integration of various external solutions with Kyma. The integration is achieved using the `publish-subscribe` messaging pattern that allows Kyma to receive business Events from different solutions, enrich them, and trigger business flows using lambdas or services defined in Kyma.
 
-To learn how to write an HTTP service or Lambda in Kyma, and handle the Event Bus published Events, check the Services Programming Model [guide](013-details-service-programming-model.md) and the Lambda Programming Model [guide](../../serverless/docs/035-programming-model.md).
+To learn how to write an HTTP service or lambda in Kyma, and handle the Event Bus published Events, check the **Services Programming Model** [guide](013-details-service-programming-model.md) and the **Lambda Programming Model** [guide](../../serverless/docs/035-programming-model.md).
```
Comment (tomekpapiernik) [inline docs/event-bus/docs/001-overview-event-bus.md]: remove cross-document links please

Comment (klaudiagrz) [reply docs/event-bus/docs/001-overview-event-bus.md]: It's not the scope of my task, but sure, I can do this.

Comment (bszwarc) [reply docs/event-bus/docs/001-overview-event-bus.md]: Ah :O

**Comment by klaudiagrz** [comment]
@tomekpapiernik 
I've provided the requested changes, can you approve please?

**Pull request merged**